frontend/src/components/pages/Default.tsx

```diff
@@ -68,8 +35,12 @@ const Default = (): React.ReactElement => {
           width="100%"
           spacing={5}
         >
-          {courses?.map((course) => (
-            <CourseCard key={course.id} />
+          {courses?.map((course: CourseResponse) => (
+            <CourseCard
+              key={course.id}
+              course={course}
+              hidden={!authenticatedUser}
```

Comment:
Could you remove this functionality? We want anyone to be able to browse courses, eg to pique their interest.